test/test_operators.py

```diff
@@ -206,9 +206,15 @@ def test_linear(self, shape, out_features):
         def linear(input, weight, bias):
             return F.linear(input, weight, bias) + 2.0
 
+        def linear_no_bias(input, weight):
```

Comment:
maybe we should test the pattern matching in python?

Reply:
Yeah sure we will need to pybind the pass to python, I will add a test

Reply:
can we avoid pybind by manually constructing a torchscript model containing t + matmul + _add and just ensuring "linear" exists in the graph_for call?